docs/admin/authorization/index.md

```diff
@@ -150,8 +150,8 @@ As of version 1.3, clusters created by kube-up.sh are configured so that the A
 
 {% endcapture %}
 {% capture whatsnext %}
-* To learn more about Authentication, see **Authentication** in [Controlling Access to the Kubernetes API](docs/admin/accessing-the-api/).
-* To learn more about Admission Control, see [Using Admission Controllers](docs/admin/admission-controllers/).
+* To learn more about Authentication, see **Authentication** in [Controlling Access to the Kubernetes API](../accessing-the-api/).
```

Comment:
Thanks for catching these. Instead of using a relative path from the current location, please use a URL w/ the root path, i.e.:

/docs/admin/accessing-the-api/

Adding a leading slash should fix the problem. Thx